src/glmfit.jl

```diff
@@ -115,7 +115,23 @@ function confint(obj::AbstractGLM, level::Real)
 end
 confint(obj::AbstractGLM) = confint(obj, 0.95)
 
-deviance(m::AbstractGLM)  = deviance(m.rr)
+deviance(m::AbstractGLM) = deviance(m.rr)
+
+function loglikelihood(m::AbstractGLM)
```

Comment:
@nalimilan The documentation for deviance should be rephrased although I'm not sure to what.  I was more-or-less following the R conventions when I wrote the code but following my incorrect understanding of deviance when I wrote the documentation for deviance.